aas_core_meta/v3_1.py

```diff
@@ -1239,6 +1193,10 @@ class Non_empty_XML_serializable_string(str, DBC):
 
         An attribute with data type "string" shall consist of these characters only:
         ``^[\x09\x0A\x0D\x20-\uD7FF\uE000-\uFFFD\u00010000-\u0010FFFF]*$``.
+        
+        Constraint AASd-130 ensures that encoding and interoperability between 
```

Comment:
I suppose this needs to go in a separate pull request as it is an unrelated change?